### PR TITLE
Add RS OCSP stapling article

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/jira-doc-1222/rs/security/certificates/ocsp-stapling/)

### CODE REVIEW

File: content/rs/security/certificates/ocsp-stapling.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+- The certificate has been revoked.
+
+- The certificate's status is unknown. This can happen if the OCSP responder fails to send a response.
+
+## Set up OCSP stapling
+
+Configure and enable OCSP stapling for your Redis Enterprise cluster with the [admin console](#admin-console-method), the [REST API](#rest-api-method), or [`rladmin`](#rladmin-method).
+
+After you enable OCSP, the server always staples the cached OCSP status. You also need to configure your client to use the stapled OCSP status.
```
Comment (kaitlynmichael) [inline content/rs/security/certificates/ocsp-stapling.md:33]: Does this mean you can't change your settings back after enabling OCSP?

**Review by kaitlynmichael — APPROVED** [review]
Looks great!

File: content/rs/security/certificates/ocsp-stapling.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/security/certificates/ocsp-stapling.md:33]: You can still turn off OCSP if you toggle the OCSP switch in the admin console or set ocsp_functionality to false with the REST API or rladmin. When ocsp_functionality equals true, the server always staples the OCSP status and sends it to the client, but it is the client's responsibility to actually use the OCSP staple. Maybe I need to reword this section a bit to clarify.

Comment (rrelledge) [reply content/rs/security/certificates/ocsp-stapling.md]: Reworded this section for clarity.

**Pull request merged**